crates/miden-lib/asm/miden/contracts/faucets/basic_fungible.masm

```diff
@@ -29,7 +29,7 @@ const.ERR_FUNGIBLE_ASSET_DISTRIBUTE_WOULD_CAUSE_MAX_SUPPLY_TO_BE_EXCEEDED="distr
 const.METADATA_SLOT=0
 
 # Basic authentication for the faucet owner.
-export.basic::auth_tx_rpo_falcon512
+export.basic::auth__tx_rpo_falcon512
```

Comment:
Do we actually need to re-export the auth procedure from here? I think this may be a remnant of one of the past designs.